helm/aws-load-balancer-controller/values.yaml

```diff
@@ -215,6 +215,9 @@ targetgroupbindingMaxConcurrentReconciles:
 # Maximum duration of exponential backoff for targetGroupBinding reconcile failures
 targetgroupbindingMaxExponentialBackoffDelay:
 
+# Enable cluster aware bindings, that will not remove targets that are not managed by the cluster
+targetgroupbindingEnableCLusterAwareBindings: 
```

Comment:
Can we make this a feature flag instead of a command line flag since Command line flags are difficult to deprecate?

Reply:
Of course, but i don't quite follow how they're implemented in this repo. Do you have an example of one?

Reply:
Here is the implementation of existing feature gates. https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/pkg/config/feature_gates.go#L14. You can reference these.